pom.xml

```diff
@@ -79,11 +79,12 @@ under the License.
         <protobuf.version>3.7.1</protobuf.version>
         <unixsocket.version>2.3.2</unixsocket.version>
         <protoc-jar-maven-plugin.version>3.11.1</protoc-jar-maven-plugin.version>
-        <flink.version>1.14.3</flink.version>
+        <flink.version>1.15.2</flink.version>
         <scala.binary.version>2.12</scala.binary.version>
         <scala.version>2.12.7</scala.version>
         <lz4-java.version>1.8.0</lz4-java.version>
-        <flink-shaded-jackson.version>2.12.4-14.0</flink-shaded-jackson.version>
+        <flink-shaded-jackson.version>2.12.4-15.0</flink-shaded-jackson.version>
+        <slf4j-log4j12.version>1.7.32</slf4j-log4j12.version>
```

Comment:
Why is this needed now?

Reply:
`org.slf4j:slf4j-log4j12` is referenced in two subprojects: `statefun-smoke-e2e-common` and `statefun-flink-distribution`, and I needed to increase the version to resolve a dependency convergence issue. So I thought I'd pull the version out to the parent pom. But I'm happy to do something else if you'd prefer.